WeakAuras/WeakAuras.lua

```diff
@@ -1414,14 +1414,18 @@ loadedFrame:SetScript("OnEvent", function(self, event, addon)
   elseif(event == "PLAYER_LOGIN") then
     local startTime = debugprofilestop()
     local finishTime = debugprofilestop()
+    local ok, msg
     -- hard limit seems to be 19 seconds. We'll do 15 for now.
     while coroutine.status(loginThread) ~= 'dead' and finishTime - startTime < 15000 do
-      coroutine.resume(loginThread)
+      ok, msg = coroutine.resume(loginThread)
       finishTime = debugprofilestop()
     end
     if coroutine.status(loginThread) ~= 'dead' then
       WeakAuras.dynFrame:AddAction('login', loginThread)
     end
+    if not ok then
```

Comment:
Is it not preferable to move this in the previous while loop ?

Reply:
A coroutine which encounters an uncaught error will terminate and have a status of dead.

So, maybe? I don't think it makes any difference either way.